### DISCUSSION TITLE
Documentation site should allow to select a specific version of iTwinUI-React

### DISCUSSION THREAD

**Discussion by HaveSpacesuit** [iTwin/iTwinUI-react#979] [Ideas]
It would be nice if the documentation site could support multiple versions of the library. Several documentation sites allow users to select the version of the tools they are working with in order to get the relevant documentation.

For example, I'm converting our BWC Wizard component to the iTwinUI-React Wizard. (My team is getting off of BWC asap, but not quite ready for iTwinUI-React 2.0). Now that the [resource page](http://itwin.github.io/iTwinUI-react) is pointing to iTwinUI-React 2.0, I am not able to find the old documentation for how to implement the older Wizard (replaced by Workflow Diagram).

**Comment by mayank99** [comment]
Thanks for the suggestion. I think we could host the v1 storybook somewhere.

Can I ask why you're not able to use 2.0 if you're migrating from bwc?

_Also I should warn that the workflow diagram was never meant to be used as a wizard. It's meant to be a "data display" component rather than one that you interact with. That's why it was deprecated._

**Reply by HaveSpacesuit** [reply]
Thanks; we don't have QA power right now to try to change from 1 to 2; after start of year we should be able to. In the meantime, we have to get rid of BWC due to security concerns.

**Reply by mayank99** [reply]
Done! See https://github.com/iTwin/iTwinUI-react/issues/1000#issuecomment-1366185466
